QgisModelBaker/gui/import_data.py

```diff
@@ -372,12 +372,11 @@ def save_configuration(self, configuration):
 
     def restore_configuration(self):
         settings = QSettings()
-
         self.fill_toml_file_info_label()
         self.xtf_file_line_edit.setText(settings.value(
             'QgisModelBaker/ili2pg/xtffile_import'))
-        self.chk_delete_data.setChecked(settings.value(
-            'QgisModelBaker/ili2pg/deleteData', False, bool))
+        # set chk_delete_data always to unchecked
```

Comment:
I am in favor of comments, where they give additional context and reasoning (and where they do not transliterate the code itself into prose).

Reply:
The reason for this comment is, because it's quite unusual at this place, that we don't read the configuration but set it just to false. So it's here to say: Hey, this is done on purpose to have the box always unchecked. 

I don't mind to remove it - it's not that important. I will remove when you tell me to without any resistance, just please accept that it's not "transliteration of the code itself into prose" :slightly_smiling_face:

Reply:
I was more hoping for a minimal bit of reasoning "set ... always to unchecked because [xxx]"
I was pretty sure that you had a good reason ;)

Reply:
"... because otherwise the user could delete the data accidentally" - okay?

Reply:
much better :+1: